extensions/applicationinsights-debugplugin-js/src/DebugPlugin.ts

```diff
@@ -327,7 +328,14 @@ export default class DebugPlugin extends BaseTelemetryPlugin {
                     }
 
                     let evtPrefix = _getEvtPrefix(funcArgs);
-                    dashboard.newLogEntry(_createInstrumentObject(funcArgs, orgArgs), dateNow() - startTime, `${evtPrefix}`, 0, funcArgs.name);
+                    let obj = _createInstrumentObject(funcArgs, orgArgs);
+                    dashboard.newLogEntry(obj, dateNow() - startTime, evtPrefix, 0, funcArgs.name);
+
+                    let dbgExt = getDebugExt(_self.core.config);
+                    if (dbgExt && dbgExt.debugMsg) {
+                        dbgExt.debugMsg(evtPrefix, obj);
+                    }
+
```

Comment:
I see this piece of code is duplicated in several places, can we extract this into a separate function?